client/src/reducers/activeMealReducer.js

```diff
@@ -0,0 +1,7 @@
+export default function(state = null, action) {
+  switch (action.type) {
+    case 'MEAL_SELECTED':
```

Comment:
It might be a good idea to change the case so it isn't a string, but a variable that references a string. This is so that if we decide to change the action type, we'll only have to change it in one place.

That is, make it something like `MEAL_SELECTED = 'MEAL_SELECTED'`. If you do change it, you'll also have to change it in the action creator and export it.